kotlinx-coroutines-core/build.gradle.kts

```diff
@@ -303,6 +304,6 @@ artifacts {
 }
 
 // Workaround for https://github.com/Kotlin/dokka/issues/1833: make implicit dependency explicit
-tasks.named("dokkaHtmlPartial") {
+tasks.withType<DokkaBaseTask>() {
     dependsOn(jvmJar)
 }
```

Comment:
Please could you try removing this workaround? It shouldn't be necessary in DGPv2.

If `./gradlew :dokkaGenerate` still complains, then please raise this.